src/robotoff.js

```diff
@@ -126,5 +126,17 @@ export default {
     return axios.get(
       `${ROBOTOFF_API_URL}/images/logos?logo_ids=${logoIds.join(",")}`
     )
+  },
+
+  getNutritionValueFromImage(imageUrl) {
+    alert(' in robotoff.getNutritionValueFromImage')
```

Comment:
```suggestion
    console.log(' in robotoff.getNutritionValueFromImage')
```

It will be easier to debug your code since `console.log()` does not block the workflow. Simply open your chrome console with <kbd>F12</kbd>